<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQModelWidget.ui,v $ </comment>
<comment>   $Revision: 1.1.2.2 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2010/10/26 14:49:54 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQModelWidget</class>
 <widget class="CopasiWidget" name="CQModelWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="mpLblNAme" >
     <property name="text" >
      <string>Model Name</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLineEdit" name="mpEditName" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="mpLblTimeUnit" >
     <property name="text" >
      <string>Time Unit</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="mpComboTimeUnit" />
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="mpLblInitialTime" >
     <property name="text" >
      <string>Initial Time</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QLineEdit" name="mpEditInitialTime" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4" >
    <widget class="Line" name="mpLine1" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="QLabel" name="mpLblCurrentTime" >
     <property name="text" >
      <string>Current Time</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" >
    <widget class="QLineEdit" name="mpEditCurrentTime" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4" >
    <widget class="Line" name="mpLine2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4" >
    <layout class="QHBoxLayout" name="mpBtnLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="mpBtnCommit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnRevert" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="mpHorizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>248</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="3" >
    <widget class="QComboBox" name="mpComboVolumeUnit" />
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="mpLblVolumeUnit" >
     <property name="text" >
      <string>Volume Unit</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QComboBox" name="mpComboAreaUnit" />
   </item>
   <item row="3" column="2" >
    <widget class="QLabel" name="mpLblAreaUnit" >
     <property name="text" >
      <string>Area Unit</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QComboBox" name="mpComboLengthUnit" />
   </item>
   <item row="4" column="2" >
    <widget class="QLabel" name="mpLblLengthUnit" >
     <property name="text" >
      <string>Length Unit</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="mpLblQuantityUnit" >
     <property name="text" >
      <string>Quantity Unit</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="mpComboQuantityUnit" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="mpLblModelType" >
     <property name="text" >
      <string>Rate Law Interpretation</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QComboBox" name="mpComboModelType" />
   </item>
   <item row="10" column="0" colspan="4" >
    <widget class="CQNotes" native="1" name="mpNotes" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQNotes</class>
   <extends>QWidget</extends>
   <header>copasi/UI/CQNotes.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpEditName</tabstop>
  <tabstop>mpComboTimeUnit</tabstop>
  <tabstop>mpComboVolumeUnit</tabstop>
  <tabstop>mpComboQuantityUnit</tabstop>
  <tabstop>mpComboAreaUnit</tabstop>
  <tabstop>mpComboModelType</tabstop>
  <tabstop>mpComboLengthUnit</tabstop>
  <tabstop>mpEditInitialTime</tabstop>
  <tabstop>mpEditCurrentTime</tabstop>
  <tabstop>mpBtnCommit</tabstop>
  <tabstop>mpBtnRevert</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnCommit</sender>
   <signal>clicked()</signal>
   <receiver>CQModelWidget</receiver>
   <slot>slotBtnOKClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>57</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnRevert</sender>
   <signal>clicked()</signal>
   <receiver>CQModelWidget</receiver>
   <slot>slotBtnRevertClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>144</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
